MIGRATION.md

```diff
@@ -281,12 +281,21 @@ The arguments of `oColorsGetContrastRatio` have been renamed. `$col1` becomes `$
 
 #### oColorsGetTint
 
-The first argument of `oColorsGetTint` has been renamed from `$color` to `$color-name`:
+Instead of `oColorsGetTint` use `oColorsGetTone`, with `$color-name` as the first argument instead of `$color`:
 
 ```diff
-$tint: oColorsGetTint('teal', 50);
+-$tint: oColorsGetTint('teal', 50);
++$tint: oColorsGetTone('teal', 50);
+
 -$tint: oColorsGetTint($color: 'teal', 50);
-+$tint: oColorsGetTint($color-name: 'teal', 50);
++$tint: oColorsGetTone($color-name: 'teal', 50);
+```
+
+`oColorsGetTint` used to fallback to a mix if a tone is not supported but `oColorsGetTone` now errors. In this case call `oColorsMix` explicitly:
```

Comment:
This is good! It will hopefully help people not mix up what a tone and a mix is.